### DISCUSSION TITLE
Contact Form

### DISCUSSION THREAD

**Discussion by loftwah** [EddieHubCommunity/LinkFree#6781] [Ideas]
Users could add a custom contact form linked to their email addresses. This could be extended to support additional functionality like integrating with Stripe for payments.

**Comment by eddiejaoude** [comment]
Interesting but a lot of work and not sure how many people will use it

**Comment by loftwah** [comment]
True. It would be a lot of work for something that could be a URL on the bio (a link to a contact form).
